pkg/scaler/httputil.go

```diff
@@ -0,0 +1,39 @@
+package scaler
+
+import (
+	"k8s.io/client-go/rest"
+	"k8s.io/klog"
+	"net/http"
+)
+
+type transportWrapper struct {
+	Transport http.RoundTripper
+}
+
+func (rt *transportWrapper) RoundTrip(req *http.Request) (*http.Response, error) {
+	klog.Infof("(transportWrapper)(RoundTrip) request: %v", req.URL.String())
```

Comment:
Should this be at a higher log level ?

Reply:
I changed it for debugging but you are right, this log should only be printed when log level is changed. Will change it to `klog.V(5).Infof` instead.

Reply:
Thanks for changing it. 
/lgtm